File: lista1/l1z2/Zad2J/app/src/main/java/com/example/zad2j/MainActivity.java
package com.example.zad2j;

import androidx.appcompat.app.AppCompatActivity;
import android.os.Bundle;
import android.view.View;
import android.widget.TextView;
import android.widget.Toast;
import java.util.Random;


public class MainActivity extends AppCompatActivity {

        public int r1=0;
        public int r2=0;
        public int number=0;
        public int n = 100;
        Random rand = new Random();
        @Override
        public void onCreate(Bundle savedInstanceState) {

                super.onCreate(savedInstanceState);
                setContentView(R.layout.activity_main);
                number = rand.nextInt(n);
                TextView RightB = (TextView) findViewById(R.id.RightButton);
                RightB.setText("It's smaller");
                TextView LeftB = (TextView) findViewById(R.id.LeftButton);
                LeftB.setText("It's higher");
                roll();
        }

        public void roll()
        {
            r1 = rand.nextInt(n-r2) + r2;
            TextView Place4number = (TextView) findViewById(R.id.Place4number);
            Place4number.setText("" + r1);

        }
        public void ButtonLeftClick(View view) {
                if(r1<=number) {
                        r2 = r1;
                        if (r1 == number) {
                        Toast.makeText(this, "Wygrales", Toast.LENGTH_SHORT).show();

                        } else {
                        Toast.makeText(this, "Dobrze", Toast.LENGTH_SHORT).show();

                        }
                }
                else
                {
                        Toast.makeText(this, "Zle", Toast.LENGTH_SHORT).show();
                }
                roll();
        }
        public void ButtonRightClick(View view) {
                if(r1>=number)
                {
                        n=r1;
                        if (r1 == number)
                        {
                                Toast.makeText(this, "Wygrales", Toast.LENGTH_SHORT).show();
                        }
                        else
                        {
                                Toast.makeText(this, "Dobrze", Toast.LENGTH_SHORT).show();
                        }
                }
                else
                {

                Toast.makeText(this, "Zle", Toast.LENGTH_SHORT).show();
                }
                roll();
        }
}

File: lista1/l1z2/Zad2/app/src/main/java/com/example/zad2/MainActivity.kt
package com.example.zad2

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.view.View
import android.widget.TextView
import android.widget.Toast
import kotlin.random.Random

class MainActivity : AppCompatActivity() {

    public var r1=0;
    public var r2=0;
    public var number=0;
    public var n = 100;
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
        number = Random.nextInt(n)
        findViewById<TextView>(R.id.RightButton).text = "It's smaller"
        findViewById<TextView>(R.id.LeftButton).text = "It's higher"
        roll()
    }

    fun roll()
    {
        r1 = Random.nextInt(n-r2) + r2
        findViewById<TextView>(R.id.Place4number).text = "" + r1
    }
    fun ButtonLeftClick(view: View) {
        if(r1<=number) {
            r2 = r1
            if (r1 == number) {
                Toast.makeText(this, "Wygrales", Toast.LENGTH_SHORT).show();

            } else {
                Toast.makeText(this, "Dobrze", Toast.LENGTH_SHORT).show();

            }
        }
        else
        {
            Toast.makeText(this, "Zle", Toast.LENGTH_SHORT).show();
        }
        roll()
    }
    fun ButtonRightClick(view: View) {
        if(r1>=number)
        {
            n=r1
            if (r1 == number) {
                Toast.makeText(this, "Wygrales", Toast.LENGTH_SHORT).show();

            } else {
                Toast.makeText(this, "Dobrze", Toast.LENGTH_SHORT).show();

            }
        }
        else
        {

            Toast.makeText(this, "Zle", Toast.LENGTH_SHORT).show();
        }
        roll()
    }
}